Automattic/jetpack
# Charts: Adding multi series support for line charts

Fixes https://github.com/Automattic/jetpack-roadmap/issues/2054

This PR depends on https://github.com/Automattic/jetpack/pull/40581

## Proposed changes:

* update line chart to accept multiple series of data
* extend existing components to accept `className` prop for more flexible styling

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* navigate to the directory and run storybook: `cd projects/js-packages/charts` and `pnpm run storybook`
* verify that the line chart now shows multiple series in the default story
* verify that existing charts are working

![SCR-20241213-kwiy](https://github.com/user-attachments/assets/485a407e-ec7b-4400-b7a1-13675f92b0c0)
